backend/app/controllers/spree/admin/orders_controller.rb

```diff
@@ -34,7 +34,7 @@ def index
           params[:q][:completed_at_lt] = params[:q].delete(:created_at_lt)
         end
 
-        @search = Order.preload(:user).accessible_by(current_ability, :index).ransack(params[:q])
+        @search = Order.preload(:user).accessible_by(current_ability, :index).ransack(params[:q].to_unsafe_h)
```

Comment:
@priyank-gupta Would it not be better to permit parameters for ransack instead of converting them to unsafe hash?